src/components/DocumentationTopic/PrimaryContent/DeclarationGroup.vue

```diff
@@ -80,6 +147,32 @@ export default {
       return this.declaration.platforms.join(', ');
     },
     isSwift: ({ interfaceLanguage }) => interfaceLanguage === Language.swift.key.api,
+    references: ({ store }) => store.state.references,
+    isExpanded: {
+      get: ({ declListExpanded }) => declListExpanded,
+      set(value) {
+        this.$emit('update:declListExpanded', value);
+      },
+    },
+  },
+  methods: {
+    async selectDeclaration(identifier) {
+      if (identifier === this.identifier) return;
+      this.selectedIdentifier = identifier;
+      // enough time to update the just selected item
+      await waitFor(100);
+      this.isExpanded = false; // collapse the list
+      // await animation to finish
+      await waitFor(500);
+      this.$router.push(this.references[identifier].url);
```

Comment:
I might be overlooking something, but I don't know if this is working as expected. When I click an alternate overload, it does change the selected UI, but it doesn't seem to update the URL and actually navigate to a new page.

Another issue I'm seeing is that I can't navigate back to the original symbol after clicking a subsequent overload. I might just be misunderstanding the expected behavior here, but not sure.

**Example**

https://github.com/apple/swift-docc-render/assets/212918/b4b7eb0e-501f-4b89-a02e-47066634b47f

Reply:
> Another issue I'm seeing is that I can't navigate back to the original symbol after clicking a subsequent overload. I might just be misunderstanding the expected behavior here, but not sure.

You can navigate back to the "generic page" with the browser back button or clicking on the navigator. It is expected that if you click on the pill, it will navigate to the "hashed" specific page.

Reply:
That sounds like what I was expecting. This works as expected with the fixture data you're using to test, but it's not working that way with the real DocC branch supporting this—you may want to work with @mayaepps to make sure that it's a known bug or incomplete functionality there and not an issue with any of the logic here—seems to be working with the ideal fixture data though. Thanks for clarifying!